config/locales/ja.yml

```diff
@@ -486,16 +487,26 @@ ja:
     setup_webauthn_html: "\U0001F389 セキュリティ機器に対応しました！新しい機器を<a href=\"/settings/edit#security-device\">設定</a>してアカウントのセキュリティを向上しましょう。<a
       href=\"https://blog.rubygems.org/2023/08/03/level-up-using-security-devices.html\">詳細はこちら</a>！"
     api:
-      mfa_required:
-      mfa_required_not_yet_enabled:
-      mfa_required_weak_level_enabled:
-      mfa_recommended_not_yet_enabled:
-      mfa_recommended_weak_level_enabled:
+      mfa_required: このgemにはMFAの有効化が必要です。まだMFAを有効にしていません。
+      mfa_required_not_yet_enabled: |
+        ［エラー］アカウントとgemの保護のため、 https://rubygems.org/totp/new で多要素認証の設定が必要です。
+
+        詳しくはブログ記事をお読みください (https://blog.rubygems.org/2022/08/15/requiring-mfa-on-popular-gems.html)。
+      mfa_required_weak_level_enabled: |
+        ［エラー］アカウントのgemの保護のため、 https://rubygems.org/settings/edit でMFAの水準を「UIとgemのサインイン」または「UIとAPI」に変える必要があります。
```

Comment:
Typo.

```diff
- アカウントのgemの保護のため
+ アカウントとgemの保護のため
```

Reply:
Applied in 82399bb863c004fec7648013a0c81aa8f3d2e841